eduart_robot/Status status
eduart_robot/Mode mode
string info_message
